### DISCUSSION TITLE
vc-graphics-polyline 切换到 2D模式 下会消失

### DISCUSSION THREAD

**Discussion by mc15829702687** [zouyaoji/vue-cesium#333] [Q&A]
vc-graphics-polyline 组件在切换到2D模式下显示不出来，在3D模式下又重新显示

![image](https://user-images.githubusercontent.com/44308726/209606159-2de8f7bf-56c0-456d-824f-93a3b3888409.png)
2D

![image](https://user-images.githubusercontent.com/44308726/209606185-9cdc1df5-6e53-4fca-bf24-fcfc8b599d0d.png)
3D

**Comment by zouyaoji** [comment]
使用在线例子测试没能重现哈。

![image](https://user-images.githubusercontent.com/24430116/209676956-3ad4615c-7c7b-4cc3-9f7f-0dd4407bb4de.png)

**Reply by mc15829702687** [reply]
大佬，我找到问题了，我给 vc-viewer 组件上加了 scene3-d-only 属性
